doc/source/icepack_index.rst

```diff
@@ -454,7 +454,7 @@ either Celsius or Kelvin units).
    "tday", "absolute day number", ""
    "Tf", "freezing temperature", "C"
    "Tffresh", "freezing temp of fresh ice", "273.15 K"
-   "tfrz_option", ":math:`\bullet` form of ocean freezing temperature", ""
+   "tfrz_option", ":math:`\bullet` form of ocean freezing temperature", "mushy"
```

Comment:
Just FYI, it's not clear whether the last column is supposed to be the default or units or something else.  I think leaving this blank is OK, it's also OK to define the default, or probably to say "character string".

Reply:
Here in the index, I think the convention is to _not_ put default values from the code unless they are hard-coded and/or are not expected to change (like the freezing temperature of fresh ice), but do put units where possible.  The default values belong in ug_case_settings, as @apcraig notes below.